### PR TITLE
Update test apps

### PR DESCRIPTION
**Pull request by MatkovIvan**

### CODE REVIEW

File: TestApps/Sasquatch/Sasquatch/Base.lproj/LaunchScreen.storyboard [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                        <rect key="frame" x="0.0" y="0.0" width="375" height="812"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="2" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="ASe-Tr-X1Z">
+                                <rect key="frame" x="57" y="456" width="261" height="41"/>
+                                <constraints>
+                                    <constraint firstAttribute="height" constant="41" id="rb3-0k-9WO"/>
+                                </constraints>
+                                <string key="text">Starting Visual Studio App Center
+Puppet Test Application
```
Comment (jaelim-ms) [inline TestApps/Sasquatch/Sasquatch/Base.lproj/LaunchScreen.storyboard]: Format

File: Puppet/Puppet/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 2]
```diff
@@ -101,6 +101,11 @@
       "idiom" : "ipad",
       "filename" : "icon-16.png",
       "scale" : "2x"
+    },
```
[the remark this answers is not in the document]
Comment (MatkovIvan) [reply Puppet/Puppet/Assets.xcassets/AppIcon.appiconset/Contents.json]: Yes

File: TestApps/Sasquatch/Sasquatch/LaunchScreen.storyboard [hunk 3]
```diff
@@ -1,51 +0,0 @@
...
-                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
-                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
-                        <subviews>
-                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" fixedFrame="YES" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="2" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="ASe-Tr-X1Z">
-                                <rect key="frame" x="30" y="444" width="315" height="41"/>
-                                <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
-                                <string key="text">Starting Visual Studio App Center
-Sasquatch Test Application
-
-</string>
```
Comment (MatkovIvan) [inline TestApps/Sasquatch/Sasquatch/LaunchScreen.storyboard]: `storyboard` doesn't manually formatted. Also, I didn't change it.
Let me know if you haven't changed your mind

File: Puppet/Puppet/Assets.xcassets/AppIcon.appiconset/Contents.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (jaelim-ms) [reply Puppet/Puppet/Assets.xcassets/AppIcon.appiconset/Contents.json]: Please remove all stuffs added by Xcode 9.

File: TestApps/Sasquatch/Sasquatch/LaunchScreen.storyboard (hunk 3 above)
Comment (jaelim-ms) [reply TestApps/Sasquatch/Sasquatch/LaunchScreen.storyboard]: Then never mind.

File: TestApps/Sasquatch/Sasquatch/Base.lproj/LaunchScreen.storyboard (hunk 1 above)
Comment (jaelim-ms) [reply TestApps/Sasquatch/Sasquatch/Base.lproj/LaunchScreen.storyboard]: This is not Puppet app. Change the name to Sasquatch.
It was initially displaying Swift/Objective C and this is why there are two launch storyboard.
I don't care of combining those, FYI.

**Pull request merged**